services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -369,31 +378,46 @@ async def _observing_single_service(service_name: str) -> None:
                 #   dy-sidecar, dy-proxy, or containers) -> it cannot be removed safely
                 # 4. service started, and failed on closing -> it cannot be removed safely
 
-                failed_while_saving_state_and_outputs = (
-                    scheduler_data.dynamic_sidecar.service_removal_state.can_save
-                    and scheduler_data.dynamic_sidecar.were_containers_created
-                )
-                if failed_while_saving_state_and_outputs:
+                if (
+                    scheduler_data.dynamic_sidecar.wait_for_manual_intervention_after_error
+                ):
                     # use-cases: 3, 4
                     # Since user data is important and must be saved, take no further
                     # action and wait for manual intervention from support.
+
+                    # After manual intervention service can now be removed
+                    # from tracking.
+
+                    modulo_divisor = _get_30_second_modulo_divisor(
+                        dynamic_scheduler.DIRECTOR_V2_DYNAMIC_SCHEDULER_INTERVAL_SECONDS
+                    )
+                    if (
+                        # trigger every ~30 seconds to reduce pressure on
+                        # docker swarm engine API.
+                        # NOTE: do not change below order
+                        self._observation_counter % modulo_divisor == 0
+                        and await is_dynamic_sidecar_stack_missing(
+                            scheduler_data.node_uuid, dynamic_sidecar_settings
+                        )
+                    ):
+                        # if both proxy and sidecar ar missing at this point it
+                        # is safe to assume that user manually removed them from
+                        # Portainer after cleaning up.
+
+                        # NOTE: saving will fail since there is no dy-sidecar,
+                        # and the save was taken care of by support. Disabling it.
+                        scheduler_data.dynamic_sidecar.service_removal_state.can_save = (
+                            False
+                        )
+                        await save_and_remove_user_created_services(
```

Comment:
does not really sound like stop observing that guy. what is this really doing?

Reply:
As already discussed the idea is to sue the same code to take care of the shutting down of the service. Let me try and rename the function.